libvast/test/system/catalog.cpp

```diff
@@ -44,7 +44,7 @@ namespace {
 
 constexpr size_t num_partitions = 4;
 constexpr size_t num_events_per_parttion = 25;
-constexpr uint32_t taste_count = 4;
+constexpr size_t taste_count = 4;
 constexpr size_t num_query_supervisors = 1;
 
```

Comment:
These 2 variables should not be necessary any more.

Reply:
Removed them in https://github.com/tenzir/vast/pull/2771/commits/d056ae26271d60fcc387da2b2b8fb661470d716d